Automattic/jetpack
# Subscriptions: Fix subscriber pop-up button text color

Fixes https://github.com/Automattic/jetpack/issues/34911

## Proposed changes:

It fixes the subscriber pop-up button text color by removing color definitions completely to use the default ones.

| Before | After |
|--------|--------|
| <img width="624" alt="Screenshot 2024-01-09 at 14 23 08" src="https://github.com/Automattic/jetpack/assets/4068554/11f38fe5-8afe-4437-82ca-55a4ac5b2e8c"> | <img width="624" alt="Screenshot 2024-01-09 at 14 22 42" src="https://github.com/Automattic/jetpack/assets/4068554/2b44a040-a8b3-45bd-abc6-a2cedbc24ae8"> | 

Tested it on some other themes and the button looks good:

#### Riverbank

<img width="624" alt="Screenshot 2024-01-09 at 14 45 27" src="https://github.com/Automattic/jetpack/assets/4068554/0624ce3e-1cbf-4fd1-8b61-d131bf923098">

#### Twenty Twenty-Two

<img width="624" alt="Screenshot 2024-01-09 at 14 46 42" src="https://github.com/Automattic/jetpack/assets/4068554/db1a3b43-eac5-4ef4-9495-ffa03c9cb110">

#### Sydney

<img width="624" alt="Screenshot 2024-01-09 at 14 50 34" src="https://github.com/Automattic/jetpack/assets/4068554/9b2ebeb2-6dac-43c0-9a21-43c8bd462632">


Not sure why sometimes there is almost no space between the input and the text above. Is this intended?

### Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:






* Have the "Enable subscriber pop-up" enabled from Newsletter settings
* Open a blog post in an incognito window
* Scroll down to open the subscriber pop-up
* Make sure the subscribe button looks good

Try to test it on a couple of different themes.
